demo/build.gradle

```diff
@@ -0,0 +1,43 @@
+plugins {
+    id "application"
+    id 'org.springframework.boot' version '2.4.4'
+    id 'io.spring.dependency-management' version '1.0.11.RELEASE'
+    id 'java'
+}
+
+apply plugin: 'idea'
+apply plugin: 'java'
+
+group 'com.amplitude'
+
+repositories {
+    jcenter()
+    mavenCentral()
+    maven {
+        url "https://plugins.gradle.org/m2/"
+    }
+}
+
+idea {
+    module {
+        sourceDirs += file('src')
+    }
+}
+
+sourceSets.main.java.srcDirs = ['src']
+
+dependencies {
+    implementation 'org.json:json:20201115'
+    implementation 'org.springframework.boot:spring-boot-starter-web'
```

Comment:
What's this package about? A package for starter?

Reply:
That's the required dependency for `Spring Web`.  A starter for building web.